server/test/controllers/applicant/ApplicantProgramBlocksControllerTest.java

```diff
@@ -494,6 +497,44 @@ public void updateFile_completedProgram_redirectsToReviewPage() {
     assertThat(result.redirectLocation()).hasValue(reviewRoute);
   }
 
+  @Test
+  public void updateFile_storedFileAlreadyExists_doesNotCreateDuplicateStoredFile() {
+    var storedFileRepo = instanceOf(StoredFileRepository.class);
```

Comment:
Not blocking: Consider adding a comment that this is a regression test and link the GitHub issue.